### PR TITLE
add test for OCF splits into multiple segments

### PR DESCRIPTION
**Pull request by Jeffxz**
Add test for this one https://github.com/w3c/epub-tests/issues/176
Tested with epubcheck 5.0 beta and 4.2 without errors.

Will add reference to specs once this is merged.

### CODE REVIEW

File: tests/ocf-zip-mult/EPUB/content_001.xhtml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>OCF ZIP container splits into segments</title>
+  </head>
+  <body>
+    <p>Test passes if reading system raised error when loading this epub file and this page of the book can not be accessed.</p>
```
Comment (iherman) [inline tests/ocf-zip-mult/EPUB/content_001.xhtml:6]:
```suggestion
    <p>Test passes if reading system raises error when loading this epub file and this page of the book can not be accessed.</p>
```

File: tests/ocf-zip-mult/EPUB/package.opf [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Open Container Format</dc:coverage>
+    <dc:creator>Zheng Xu</dc:creator>
+    <dc:date>2022-05-28</dc:date>
```
Comment (iherman) [inline tests/ocf-zip-mult/EPUB/package.opf:6]:
```suggestion
    <dc:date>2022-07-25</dc:date>
```

File: tests/ocf-zip-mult/EPUB/package.opf [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    <dc:description>MUST treat any OCF ZIP container that splits the content into segments [zip] as in error.</dc:description>
+    <dc:identifier id="pub-id">ocf-zip-mult</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:publisher>W3C</dc:publisher>
+    <dc:title>OCF ZIP container splits into segments</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="dcterms:isReferencedBy">https://w3c.github.io/epub-specs/epub33/rs/#confreq-zip-mult</meta>
+    <meta property="dcterms:modified">2022-05-28T00:00:00Z</meta>
```
Comment (iherman) [inline tests/ocf-zip-mult/EPUB/package.opf:15]:
```suggestion
    <meta property="dcterms:modified">2022-07-25T00:00:00Z</meta>
```

**Review by iherman — APPROVED** [review]

File: tests/ocf-zip-mult/EPUB/package.opf [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Open Container Format</dc:coverage>
+    <dc:creator>Zheng Xu</dc:creator>
+    <dc:date>2022-05-28</dc:date>
+    <dc:description>MUST treat any OCF ZIP container that splits the content into segments [zip] as in error.</dc:description>
```
Comment (iherman) [inline tests/ocf-zip-mult/EPUB/package.opf:7]:
```suggestion
    <dc:description>MUST treat any OCF ZIP container that splits the content into segments as in error.</dc:description>
```

File: tests/ocf-zip-mult/EPUB/content_001.xhtml (hunk 1 above)
Comment (Jeffxz) [reply tests/ocf-zip-mult/EPUB/content_001.xhtml:6]: Thanks, will fix this one and others later today

Comment (Jeffxz) [reply tests/ocf-zip-mult/EPUB/content_001.xhtml]: updated

File: tests/ocf-zip-mult/EPUB/package.opf (hunk 2 above)
Comment (Jeffxz) [reply tests/ocf-zip-mult/EPUB/package.opf]: updated

File: tests/ocf-zip-mult/EPUB/package.opf (hunk 3 above)
Comment (Jeffxz) [reply tests/ocf-zip-mult/EPUB/package.opf]: updated

File: tests/ocf-zip-mult/EPUB/package.opf (hunk 4 above)
Comment (Jeffxz) [reply tests/ocf-zip-mult/EPUB/package.opf]: updated, thanks

**Comment by iherman** [comment]
Thanks @Jeffxz. I know that Dan is very busy these days, I will therefore merge this now. Can you take care of the PR in the spec repo for the anchors?

**Comment by Jeffxz** [comment]
THanks @iherman. Will create PR to spec repo later today.

**Pull request merged**